src/components/VImageDetails/VImageDetails.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <section class="w-full">
+    <div class="flex flex-row justify-between items-center mb-6">
+      <h3 class="text-2xl md:text-3xl">
+        {{ $t('image-details.information.title') }}
+      </h3>
+      <span
+        class="flex flex-row justify-end items-center text-sr text-dark-charcoal-70 font-semibold"
+      >
+        <span class="hidden md:inline">{{
+          $t('image-details.report.large')
+        }}</span>
+        <span class="md:hidden">{{ $t('image-details.report.short') }}</span>
+        <FlagSvg class="ms-2"
+      /></span>
+    </div>
+    <dl>
+      <div>
+        <dt>{{ $t('image-details.information.type') }}</dt>
+        <dd class="uppercase">{{ imgType }}</dd>
+      </div>
+      <div v-if="providerName != sourceName">
+        <dt>{{ $t('image-details.information.provider') }}</dt>
+        <dd>{{ providerName }}</dd>
+      </div>
+      <div>
+        <dt>{{ $t('image-details.information.source') }}</dt>
+        <dd>
+          <a
+            :href="image.foreign_landing_url"
+            target="blank"
+            rel="noopener noreferrer"
+            class="text-pink"
+            >{{ sourceName }}</a
+          >
+        </dd>
+      </div>
+      <div>
+        <dt>{{ $t('image-details.information.dimensions') }}</dt>
+        <dd>
+          <!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
+          {{ imageWidth }} &times; {{ imageHeight }}
+          {{ $t('image-details.information.pixels') }}
+        </dd>
+      </div>
+    </dl>
+  </section>
+</template>
+
+<script>
+import { computed, defineComponent, useContext } from '@nuxtjs/composition-api'
+
+import FlagSvg from '~/assets/icons/flag.svg?inline'
+
+const VImageDetails = defineComponent({
+  name: 'VImageDetails',
+  components: { FlagSvg },
+  props: {
+    image: {
+      type: Object,
+      required: true,
+    },
+    imageWidth: {
+      type: Number,
+    },
+    imageHeight: {
+      type: Number,
+    },
+    imageType: {
+      type: String,
+    },
+  },
+  setup(props) {
+    const { store } = useContext()
+
+    const imgType = computed(() => {
+      if (props.imageType) {
+        if (props.imageType.split('/').length > 1) {
+          return props.imageType.split('/')[1].toUpperCase()
+        }
+        return props.imageType
+      }
+      return 'Unknown'
```

Comment:
Does this need a translation or is it handled elsewhere?

Reply:
Needs translation, good catch! Fixed in [`7900ba7`](https://github.com/WordPress/openverse-frontend/pull/682/commits/7900ba79d81e406d2114fe4148e88e392d73f177).